### ISSUE TITLE
Design: finalize color scheme

### ISSUE THREAD

**Issue by sunsplat** [womencodersnet/wcn-website#27]
Designers finalize color scheme to use for website

Acceptance Criteria (AC):
Finalize color scheme and provide HTML codes and palette for the colors
Create diagram for when to apply the colors (links, visited links, background colors, text, etc)
Share color scheme with website team

**Comment by sunsplat** [comment]
Once this is completed, we can create tasks for updating the website to follow the color scheme

**Comment by sunsplat** [comment]
Figma design located here: https://www.figma.com/design/H3tS269chkMiR0aWRrFtBt/LandingPage?node-id=0-1&p=f

@hernandezforges I feel like you had showed some color palettes at our last website meeting. Do you know the HTML codes for them or where to find them?

**Comment by hernandezforges** [comment]
@sunsplat Yes, I did. The palette is called [Peacock Feathers](https://www.color-hex.com/color-palette/11109). I've included it in the [pinned post](https://discord.com/channels/1230709748790333550/1268665247041060964), Google Drive, and our [meeting notes](https://docs.google.com/document/d/1aHrJYVbv3DdjtajKyeNW-pMreyvAVSW9WmfFhoRGZqo/edit?usp=sharing). Let me know if I need to include it anywhere else.

**Comment by sunsplat** [comment]
@hernandezforges  Awesome! Thanks!
